packages/ui-brand/scripts/start.js

```diff
@@ -0,0 +1,3 @@
+console.log(
+  "`yarn start` doesn't work here. Maybe you meant to run `yarn storybook` at project root?"
+);
```

Comment:
If these commands are just going to log a line, you can use an echo directly in the package.json file.

```json
"scripts": {
  "start": "echo '`yarn start` doesn\'t work here. Maybe you meant to run `yarn storybook` at the project root?'"
}
```
```

Reply:
Will be addressed in #1208